composer.json

```diff
@@ -19,41 +19,41 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "knplabs/knp-snappy-bundle": "^1.7",
         "myclabs/php-enum": "^1.7",
         "php-http/message-factory": "^1.1",
-        "sylius/resource-bundle": "^1.6",
-        "sylius/sylius": "~1.11.4 || ~1.12.0",
```

Comment:
We should mention in the UPGRADE file the removal of support for Sylius 1.11 and PHP 8.0